src/api/spec/features/webui/maintenance_workflow_spec.rb

```diff
@@ -1,8 +1,4 @@
 require 'browser_helper'
-# WARNING: If you change tests make sure you uncomment this line
```

Comment:
would you mind removing this warning from all specs then?

Reply:
This only fixes feature test, so nope.

Reply:
I see